src/lib/framework/Services.php

```diff
@@ -391,4 +391,9 @@ final class Services
     ///////////////////////////////////////////////////////////////
 
     public const LDAP_LOGGER = 'ldap.logger';
+
+    /**
+     * @see \OrangeHRM\LDAP\Service\LDAPService
+     */
+    public const LDAP_SERVICE = 'ldap.ldap_service';
```

Comment:
if this is not necessary, remove it. I didn't add it cz may lead to bugs, using this service. cz based on the config need to create new instance.
One reason is test ldap service.

Reply:
removed